### ISSUE TITLE
Overlay not showing

### ISSUE THREAD

**Issue by Kh4r4** [briansd9/exile-diary#75]
Since last update the overlay is not showing anymore (tracker is working in the app)

Exile Diary is running as Admin
PoE is in windowed Full Screen

Can include log if needed

Thanks!

**Comment by briansd9** [comment]
Yes, can you send me your log file please? Thanks.

Also, are you using Windows 10?

**Comment by Kh4r4** [comment]
Here it is:
[log1.txt](https://github.com/briansd9/exile-diary/files/4943652/log1.txt)

Yes, I am using Windows 10 (v2004 19041.388)

**Comment by Kh4r4** [comment]
Hello, just a quick update: the issue is inconsistent as the overlay is sometimes showing for 1 session, and sometimes not. Restarting both PoE and Exile Diary does not change anything.

**Comment by briansd9** [comment]
Yes, I've encountered the same, overlay only works sometimes... still looking for the cause.

Do you have an antivirus program running on your computer?
